plugins/filesystem/local/adapter/adapter.php

```diff
@@ -12,6 +12,9 @@
 JLoader::import('joomla.filesystem.file');
 JLoader::import('joomla.filesystem.folder');
 
+require_once 'filenotfoundexception.php';
```

Comment:
Anyway if these are common to cloud filesystems
Why we don't use this plugin as the one who really deals with files, like PlgFileSystemGoogleDrive does not need to know how a file handled in Local File System, it is encapsulated on this plugin, it can provide the functionality without a change.

Any comments about that approach ?
I moved those keeping that design in my mind